--- a/src/platform/linux/wayland/window.rs
+++ b/src/platform/linux/wayland/window.rs
@@ -389,6 +389,7 @@
                     state.tiling = configure.tiling;
                     // Limit interactive resizes to once per vblank
                     if configure.resizing && state.resize_throttle {
+                        state.xdg_surface.ack_configure(serial);
                         return;
                     } else if configure.resizing {
                         state.resize_throttle = true;
